src/lib/y2network/dialogs/add_interface.rb

```diff
@@ -23,18 +24,13 @@ def run
         ret = super
         log.info "AddInterface result #{ret}"
         ret = :back if ret == :abort
-        # TODO: replace with builder initialization
-        if ret == :back
```

Comment:
for now you cannot remove it unless you also remove that LanItems.AddItem otherwise it crash. Just try it yourself - click on AddDevice and click on Cancel.

Reply:
ah I still live in the other PR where Items are not used.

Reverted